azure-ipam/main.go

```diff
@@ -21,7 +21,7 @@ func main() {
 
 func executePlugin() error {
 	// logger config
-	var loggerCfg *logger.Config
+	loggerCfg := &logger.Config{}
```

Comment:
maybe inline setting the opts from right below?
```suggestion
	loggerCfg := &logger.Config{
		Level: 		  "debug",
		OutputPaths: 	  "stdout",
		ErrorOutputPaths: "stderr"
	}
```